.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 10
```

Comment:
That seems a tad unforgiving. We have a fair amount of issues that would take a lot of work to fix. I suggest maybe 30 here, just to give people more time to react, but at least 14 so that we have a couple weeks to mention it at dev meetings and discuss it.